### PR TITLE
Add New Hello World Skins to Translation Pipeline

### PR DESCRIPTION
**Pull request by epeach**
Adding the new Hello World Skins to the translation pipeline so that we can begin the translation process.

cc: @castro-jorge (so you'll have insight when this is merged)

### CODE REVIEW

**Review by hannahbergam — APPROVED** [review]
LGTM! I've never added skins to the translation pipeline here (only through adding words to the en_us file), but I'm learning!

**Review by bethanyaconnor — APPROVED** [review]
🎉

File: lib/cdo/script_constants.rb [hunk 1]
```diff
@@ -297,6 +297,8 @@ module ScriptConstants
     HELLO_WORLD_ANIMALS = 'hello-world-animals'.freeze,
     HELLO_WORLD_EMOJI = 'hello-world-emoji'.freeze,
     HELLO_WORLD_RETRO = 'hello-world-retro'.freeze,
+    HELLO_WORLD_SPACE = 'hello-world-space'.freeze,
```
Comment (bethanyaconnor) [inline lib/cdo/script_constants.rb]: oh actually, I think these should be in the `hoc` section of this list with `hello-world-food-2021` and the other units from last hoc

Comment (bethanyaconnor) [reply lib/cdo/script_constants.rb]: Sorry -- the unit names from last year kept changing and we kept adding "temporary" scripts to this file. Looks like we missed cleaning a few of these up :(

Comment (epeach) [reply lib/cdo/script_constants.rb]: Updated! I looked at removing the 'hello-world-retro'/emoji/animals/food, but it looks like those scripts do exist separately from the 'hello-world-retro-2021' scripts, so I left them to err on caution. Let me know if we should also remove those!

**Comment by epeach** [comment]
@bethanyaconnor PTAL! Thank you!

File: lib/cdo/script_constants.rb (hunk 1 above)
Comment (bethanyaconnor) [reply lib/cdo/script_constants.rb]: oh interesting I thought we removed them so I misremembered that! But definitely no need to clean them up right now

**Review by bethanyaconnor — APPROVED** [review]
LGTM!

**Pull request merged**